Library/Homebrew/cask/audit.rb

```diff
@@ -290,6 +297,24 @@ def check_download
       add_error "download not possible: #{e.message}"
     end
 
+    def check_appcast_contains_version
+      return unless check_appcast?
+      return if cask.appcast.to_s.empty?
+
+      appcast_stanza = cask.appcast.to_s
+      appcast_contents, = curl_output("--max-time", "5", appcast_stanza)
+      version_stanza = cask.version.to_s
+      adjusted_version_stanza = version_stanza.split(",")[0].split("-")[0].split("_")[0]
+      if appcast_contents.include? adjusted_version_stanza
+        true
+      else
+        w2 = "does not contain the version number:"
```

Comment:
Remove this variable, you can split the string on multiple lines like this:

```ruby
"single" \
"line" \
"string"

```

Reply:
thanks